flow/raster_cache_unittests.cc

```diff
@@ -139,19 +147,22 @@ TEST(RasterCache, DeviceRectRoundOut) {
 
   SkMatrix ctm = SkMatrix::MakeAll(1.3312, 0, 233, 0, 1.3312, 206, 0, 0, 1);
 
+  SkCanvas canvas(100, 100, nullptr);
+  canvas.setMatrix(ctm);
+
   sk_sp<SkColorSpace> srgb = SkColorSpace::MakeSRGB();
   ASSERT_FALSE(
       cache.Prepare(NULL, picture.get(), ctm, srgb.get(), true, false));
-  ASSERT_FALSE(cache.Get(*picture, ctm).is_valid());
+  ASSERT_FALSE(cache.Draw(*picture, canvas));
   cache.SweepAfterFrame();
   ASSERT_TRUE(cache.Prepare(NULL, picture.get(), ctm, srgb.get(), true, false));
-  ASSERT_TRUE(cache.Get(*picture, ctm).is_valid());
+  ASSERT_TRUE(cache.Draw(*picture, canvas));
 
-  SkCanvas canvas(100, 100, nullptr);
-  canvas.setMatrix(ctm);
   canvas.translate(248, 0);
-
-  cache.Get(*picture, ctm).draw(canvas);
+#ifndef SUPPORT_FRACTIONAL_TRANSLATION
+  canvas.setMatrix(RasterCache::GetIntegralTransCTM(canvas.getTotalMatrix()));
+#endif
+  cache.Draw(*picture, canvas);
```

Comment:
Would it make sense to ASSERT_TRUE here? If someone makes a change that causes this instance to fail to use the cache then we won't be testing what we came here to test because the `return true` will happen before the call tries to draw the cached image.